### PR TITLE
Fix bug on update chart, and add proptypes for pipSize

### PR DESCRIPTION
**Pull request by qingweibinary**
The non-react example is still not working, guess I have some misunderstanding on UMD

### CODE REVIEW

File: src/config/updateChart.js [hunk 1]
```diff
@@ -33,4 +33,19 @@ export default (chart, prevProps, nextProps) => {
         const { tradingTimes } = nextProps;
         updateTradingTimes({ chart, tradingTimes });
     }
+
+    if (nextProps.pipSize && (prevProps.pipSize !== nextProps.pipSize)) {
```
Comment (borisyankov) [inline src/config/updateChart.js]: Noooooooo

File: non-react-example.html [hunk 2]
```diff
@@ -3,12 +3,16 @@
 <head>
     <meta charset="UTF-8">
     <title>Non React Test</title>
-    <script src="https://npmcdn.com/binary-utils@1.8.0/lib/binary-live-api.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.2.0/require.js"></script>
```
Comment (borisyankov) [inline non-react-example.html]: Nope, we definitely do not need that.

File: non-react-example.html [hunk 3]
```diff
@@ -3,12 +3,16 @@
...
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/react/15.0.2/react.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/react/15.0.2/react-dom.min.js"></script>
+    <script src="lib/binary-charts.js"></script>
 </head>
 <body>
     <div id="chart"></div>
     <script>
-        Chart = BinaryChart.createChart('chart', { ticks: [] });
+        var BinaryCharts = require('binary-charts');
```
Comment (borisyankov) [inline non-react-example.html]: It should be available as global, so no need for the require.

File: non-react-example.html [hunk 4]
```diff
@@ -3,12 +3,16 @@
 <head>
     <meta charset="UTF-8">
     <title>Non React Test</title>
-    <script src="https://npmcdn.com/binary-utils@1.8.0/lib/binary-live-api.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.2.0/require.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/react/15.0.2/react.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/react/15.0.2/react-dom.min.js"></script>
```
Comment (borisyankov) [inline non-react-example.html]: We'd better do a production release.

File: src/config/updateChart.js (hunk 1 above)
Comment (qingweibinary) [inline src/config/updateChart.js]: why ?

**Pull request closed without merging**